mirrulations-dashboard/src/mirrdash/static/index.js

```diff
@@ -97,21 +94,26 @@ const updateClientDashboardData = () => {
             num_jobs_comments_queued,
             num_jobs_dockets_queued,
             num_jobs_documents_queued,
+            regulations_total_dockets,
+            regulations_total_documents,
+            regulations_total_comments
         } = jobInformation;
+
+        let regulations_totals = regulations_total_dockets + regulations_total_documents + regulations_total_comments
+
         updateHtmlValues(num_jobs_waiting, num_jobs_done);
-        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done, num_attachments_done], [232255, 1718669, 18072106, 15000000]); //TO DO: change hard coded numbers
+        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done, num_attachments_done], regulations_totals)
         // Counts for percents
-        updateJobTypeProgress("dockets-done",num_dockets_done, 232255);
+        updateJobTypeProgress("dockets-done", num_dockets_done, regulations_total_dockets);
+        updateJobTypeProgress("documents-done",num_documents_done, regulations_total_documents);
+        updateJobTypeProgress("comments-done",num_comments_done, regulations_total_comments);
         // Current estimate of number of attachments (from comments)
         updateJobTypeProgress("attachments-done",num_attachments_done, 15000000); 
```

Comment:
I had an idea for this 
if we did
`(num_attachments_done / num_comments_done)* regulations_total_comments`
for `total_attachments`

Reply:
Yeah, good idea, that's added in. It displays NaN in a local dev environment (divide by 0) but I figured that's a better default value than 0 or an arbitrary number.

Reply:
Works Properly when not dividing by 0! Good call guys.